Dockerfile

```diff
@@ -32,6 +32,11 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
     autoconf \
     libglpk-dev
 
+# AWS
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.0.30.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install
```

Comment:
Looks consistent with the instructions here: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html !